credit-conveyor/Deal/src/main/java/com/deals/controller/LoanApplicationController.java

```diff
@@ -0,0 +1,37 @@
+package com.deals.controller;
+
+import com.deals.dto.LoanApplicationRequestDTO;
+import com.deals.dto.LoanOfferDTO;
+import com.deals.dto.ScoringDataDTO;
+import com.deals.service.DealServices;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RequestMapping("/api/deal")
+public class LoanApplicationController {
+
+    @Autowired
+    DealServices dealServices;
```

Comment:
Autowired on field does't recommended, because it has low perfomance. Use `private final` modificator + `@RequiredArgsConstructor` from Lombok instead.